### DISCUSSION TITLE
Need help recreating the demo video workflow

### DISCUSSION THREAD

**Discussion by aksh1618** [mikesmithgh/kitty-scrollback.nvim#192] [Q&A] [answered]
Thanks for this cool plugin! This was exactly what I needed in my workflow, thanks a ton for creating it!

I'm trying to do something similar to the workflow shown in the [main demo video](https://github.com/mikesmithgh/kitty-scrollback.nvim/wiki/kitty_scrollback_nvim). So far I've managed to make it work to the point where whenever I select something in visual mode and yank with `y`, the paste window shows up momentarily, before disappearing, and then the contents are pasted to Kitty. However, from the video (and the docs etc.) it looks like the paste window shouldn't disappear (The same behavior happens even if I do `yy` in normal mode).

To solve this, I tried to yank to some random register, upon which it didn't exit but also didn't open the paste window. So I used `i` to make the paste window appear, and pasted with `<C-S-v>`. But then I couldn't figure out how to go to normal mode inside the paste window without exiting it (tried `<Esc>` as well as `<C-c>`), so was unable to edit the pasted lines.

At this point I stopped, as it seems like a case of misconfiguration at my end. Could you please share the configuration required to make the exact workflow shown in the main video work?

Here is my configuration in lazy:
```kitty.lua
return {
        -- open kitty scrollback in nvim
        "mikesmithgh/kitty-scrollback.nvim",
        enabled = true,
        lazy = true,
        cmd = { "KittyScrollbackGenerateKittens", "KittyScrollbackCheckHealth" },
        event = { "User KittyScrollbackLaunch" },
        -- version = '*', -- latest stable version, may have breaking changes if major version changed
        version = "^4.0.0", -- pin major version, include fixes and features that do not have breaking changes
        config = function()
            require("kitty-scrollback").setup()
        end,
}
```

And in kitty I have:

```kitty.conf
#: Nvim-Scrollback {{{

# allow controlling via socket
allow_remote_control socket-only
listen_on unix:/tmp/kitty
# allow browsing history
shell_integration enabled
# kitty-scrollback.nvim Kitten alias
action_alias kitty_scrollback_nvim kitten /home/aksh1618/.local/share/nvim/lazy/kitty-scrollback.nvim/python/kitty_scrollback_nvim.py
# Browse scrollback buffer in nvim
map kitty_mod+h kitty_scrollback_nvim
# Browse output of the last shell command in nvim
map kitty_mod+g kitty_scrollback_nvim --config ksb_builtin_last_cmd_output
# Show clicked command output in nvim
mouse_map ctrl+shift+right press ungrabbed combine : mouse_select_command_output : kitty_scrollback_nvim --config ksb_builtin_last_visited_cmd_output

#: }}}
```

**Comment by mikesmithgh** [comment, marked as answer]
Hey @aksh1618 thanks for letting me know. 

> whenever I select something in visual mode and yank with y, the paste window shows up momentarily, before disappearing, and then the contents are pasted to Kitty.

This sounds like a bug. My best guess is that you may have some keymaps that are conflicting with kitty-scrollback.nvim.

Could you try updating the `action_alias` line in `kitty.conf` to:
```
action_alias kitty_scrollback_nvim kitten /home/aksh1618/.local/share/nvim/lazy/kitty-scrollback.nvim/python/kitty_scrollback_nvim.py --nvim-args --clean --noplugin -n
```
Then reload your Kitty config with `ctrl+shift+f5` (`⌃ + ⌘ + ,` on macOS)

The args `--nvim-args --clean --noplugin -n` will open a clean nvim without your config or plugins.

Please let me know if this works. If this works, then we can try to hunt down the conflicting keymaps.

**Comment by aksh1618** [comment]
Thanks for pointing me in the right direction! It indeed worked with a clean nvim, and the I managed to figure out the culprit: It was an [autosave plugin](https://github.com/Pocco81/auto-save.nvim). I tried disabling it based on `vim.env.KITTY_SCROLLBACK_NVIM` but it didn't work, even though if I manually disable it in kitty scrollback using `:ASToggle`, it works fine. So for now I'm going to disable the plugin until I figure out another way to automatically disable it (maybe via an autocmd). Thanks again for your help!

**Reply by mikesmithgh** [reply]
@aksh1618 thank you for the details, I created an issue for this https://github.com/mikesmithgh/kitty-scrollback.nvim/issues/196

If you come up with any configuration that works for you, could you post to that issue?

Thanks again!

**Reply by aksh1618** [reply]
Sure, will do!
